addons/stock/models/stock_pack_operation.py

```diff
@@ -233,18 +233,18 @@ def _check_serial_number(self):
                 if operation.product_id.tracking == 'serial':
                     for opslot in operation.pack_lot_ids:
                         if opslot.qty not in (1.0, 0.0):
-                            raise UserError(_('You should provide a different serial number for each piece'))
+                            raise UserError(_('You should provide a different lot/serial number for each piece'))
```

Comment:
I don't think this error is about the lots. I think it concerns a serial number restriction.